src/libostree/ostree-repo-pull.c

```diff
@@ -4060,6 +4101,62 @@ ostree_repo_pull_with_options (OstreeRepo             *self,
       ostree_async_progress_set_status (pull_data->progress, buf->str);
     }
 
+  if (bytes_transferred > 0)
+    {
+#ifdef HAVE_LIBSYSTEMD
```

Comment:
This will fail to compile on non-systemd platforms, no? Maybe just wrap the whole if-statement instead?